### DISCUSSION TITLE
State as ONLINE when cabin overheat temp range

### DISCUSSION THREAD

**Discussion by juannicolas** [teslamate-org/teslamate#2953] [General]
Yesterday I left my car under the sun few hours, I checked the battery consumption during that time and for some reason from 11Am to 5PM appears my car was ONLINE state. I checked other application I have, and it says my car was under cabin overheat protection due to the high heat level. The funny part here is that I have the cabin protection off since I got my car.

![Screenshot 2022-10-23 230054](https://user-images.githubusercontent.com/597411/197441876-3d3371b1-9398-4a42-bca6-30aa88725060.png)

Other app telling me that my car was under cabin protection mode

![Screenshot_20221023-225603](https://user-images.githubusercontent.com/597411/197441989-9e84f4c0-d3e6-47d8-aab1-d05f47f97dd8.jpg)

When this option is OFF in the app

![Screenshot_20221023-225538_Tesla](https://user-images.githubusercontent.com/597411/197442041-d3054b57-d651-4050-8676-83afd5a55dca.jpg)

**Comment by BabyYeggie** [comment]
That seems normal to me. As soon as the interior hits 100F/40C, my 3 comes online regardless of what the cabin overheat is set to. The No A/C doesn't seem to do much, so I changed it to "On" since online mode is wasting energy anyway. 

I'm wondering if it's some sort of safety trigger, waiting for you to vent the windows (which do nothing) or switch to No A/C mode (which also does nothing in my climate).
